README.tfdoc.hcl

```diff
@@ -0,0 +1,644 @@
+          type        = string
+          description = <<-END
+            The attribute to use as the hash (partition) key. Must also be defined as an attribute, see below. Forces new resource.
+          END
+        }
+
+        variable "billing_mode" {
+          type        = string
+          default     = "PROVISIONED"
+          description = <<-END
+            Controls how you are charged for read and write throughput and how you manage capacity.
+            The valid values are `PROVISIONED` and `PAY_PER_REQUEST`.
+          END
+        }
+
+        variable "range_key" {
+          type        = string
+          description = <<-END
+            The attribute to use as the range (sort) key. Must also be defined as an attribute, see below. Forces new resource.
+          END
+        }
+
+        variable "write_capacity" {
+          type        = number
+          default     = 5
+          description = <<-END
+            The number of write units for this table. If the billing_mode is PROVISIONED, this field is required.
+          END
+        }
+
+        variable "read_capacity" {
+          type        = number
+          default     = 5
+          description = <<-END
+            The number of read units for this table. If the billing_mode is PROVISIONED, this field is required.
+          END
+        }
+
+        variable "attributes" {
+          required    = true
+          type        = map(string)
+          description = <<-END
+            List of nested attribute definitions. Only required for hash_key and range_key attributes.
+
+            ```
+            hash_key = "LockID"
+
+            attributes = {
+              LockID = "S"
+            }
+            ```
+
+            **A note about attributes**
+
+            Only define attributes on the table object that are going to be used as:
+
+            - Table hash key or range key
+            - LSI or GSI hash key or range key
+
+            The DynamoDB API expects attribute structure (name and type) to be passed along when creating or updating GSI/LSIs or creating the initial table. In these cases it expects the Hash / Range keys to be provided; because these get re-used in numerous places (i.e the table's range key could be a part of one or more GSIs), they are stored on the table object to prevent duplication and increase consistency. If you add attributes here that are not used in these scenarios it can cause an infinite loop in planning.
+          END
+
+          attribute "name" {
+            required    = true
+            type        = string
+            description = <<-END
+              The name of the attribute
+            END
+          }
+
+          attribute "type" {
+            required    = true
+            type        = string
+            description = <<-END
+              Attribute type, which must be a scalar type: S, N, or B for (S)tring, (N)umber or (B)inary data
+            END
+          }
```

Comment:
there is no attribute `name` or `type` here.... it's just a `map(string)` see the example.

```suggestion
```